edge/pkg/edged/edged_status.go

```diff
@@ -348,6 +351,11 @@ func (e *edged) setMemInfo(total, allocated v1.ResourceList) error {
 	mem := resource.MustParse(strconv.FormatInt(totalMem, 10) + "Mi")
```

Comment:
The variables name of this func is awful. Change 

- `total` to `capcility`, 
- `allocated` to `allocatable`, 
- `m` to `machineMem`, 
- `totalMem ` to `machineMemKi`, 
- `mem` to `machineMemMi`